modules/library/api/src/test/java/org/geotools/styling/AnchorPointTest.java

```diff
@@ -0,0 +1,36 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ *
+ *    (C) 2017, Open Source Geospatial Foundation (OSGeo)
+ *
+ *    This library is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation;
+ *    version 2.1 of the License.
+ *
+ *    This library is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ */
+package org.geotools.styling;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+/**
+ *
+ *
+ * @source $URL$
+ */
+public class AnchorPointTest {
+
+  @Test
+  public void testDefaultAnchorPoint() {
+    AnchorPoint anchorPoint = AnchorPoint.DEFAULT;
+
+    assertEquals(0.0, anchorPoint.getAnchorPointX().evaluate(null, Double.class), 0.0);
+    assertEquals(0.5, anchorPoint.getAnchorPointY().evaluate(null, Double.class), 0.0);
+  }
```

Comment:
The expected values are 0.0 for x and 0.5 for y. The 0.0 at the end of those lines are the delta values for float comparisons.